platform/ios/src/MGLMapView.mm

```diff
@@ -387,10 +387,9 @@ - (void)commonInit
     self.accessibilityTraits = UIAccessibilityTraitAllowsDirectInteraction | UIAccessibilityTraitAdjustable;
     _accessibilityCompassFormatter = [[MGLCompassDirectionFormatter alloc] init];
     _accessibilityCompassFormatter.unitStyle = NSFormattingUnitStyleLong;
-
     self.backgroundColor = [UIColor clearColor];
     self.clipsToBounds = YES;
-
+    if (@available(iOS 11.0, *)) { self.accessibilityIgnoresInvertColors = YES; }
```

Comment:
Did we originally set `accessibilityIgnoresInvertColors` on each individual subview to avoid affecting annotation views and callouts? Or was it related to the use of `-respondsToSelector:`?

Reply:
It was to allow users to set `accessibilityIgnoresInvertColors` on individual subviews, but I don't think that's necessary. With this implementation, annotation views and callouts seem to be ignored.